### ISSUE TITLE
AI generation of CRUD by entity

### ISSUE THREAD

**Issue by Noob-A** [AscenderTeam/AscenderFramework#38]
implementing in v1.2.6

**Comment by Noob-A** [comment]
go over endpoints, repository, service and show example for simple crud + user query

**Comment by Zahgrom34** [comment]
Good idea, but also bad idea, this would be an overhead for simple framework and the best solution is to make a specific addon or plugin (e.g. pypi module) that expands Ascender CLI's capabilities
